### PR TITLE
analog initial app v1.5.0

### PR DESCRIPTION
**Pull request by multignite**
### v1.5.0
- changed `"vite": "~5.2.11",` to avoid a minor temp analog error 
- everything else is completely standard 

- [x] Initial analog app

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+.idea/
+.project
+.classpath
+.c9/
+*.launch
+.settings/
+*.sublime-workspace
+
+# Visual Studio Code
+.vscode/*
```
Comment (Idrinth) [inline .gitignore:24]: why are the ones below  not ignored?

File: angular.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          }
+        },
+        "test": {
+          "builder": "@analogjs/platform:vitest"
+        }
+      }
+    }
+  },
+  "cli": {
+    "analytics": "4c4f9b4e-9dcc-4af0-a526-b89dc2761a04"
```
Comment (Idrinth) [inline angular.json:55]: what is this?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "translation",
+  "version": "0.0.0",
+  "type": "module",
+  "engines": {
+    "node": ">=18.19.1"
```
Comment (Idrinth) [inline package.json]: we only support 20+

**Review by Idrinth — CHANGES_REQUESTED** [review]

File: angular.json (hunk 2 above)
Comment (multignite) [reply angular.json:55]: that's me agreeing to share data with the angular teem for improvements, I usually go with `yes` but I can disable it, makes no difference for this case

File: package.json (hunk 3 above)
Comment (multignite) [reply package.json]: ok will have to try and up it and see how that goes

File: .gitignore (hunk 1 above)
Comment (multignite) [reply .gitignore:24]: that one is a bit unusual, auto generated, but still a bit unusual, will look into it

Comment (multignite) [reply .gitignore:24]: ✅

File: package.json (hunk 3 above)
Comment (multignite) [reply package.json]: ✅

**Review by Idrinth — APPROVED** [review]

**Pull request merged**